lib/requester/core-body-builder.js

```diff
@@ -154,18 +154,13 @@ formDataBodyReducer = function (data, param) {
         formParam.value = E; // make sure value is not null/undefined ever
     }
 
-    // if data has a truthy content type, we mutate the value to take the options. we are assuming that
-    // blank string will not be considered as an accepted content type.
     if (param.contentType && typeof param.contentType === STRING) {
         (options || (options = {})).contentType = param.contentType;
     }
 
-    // additionally parse the file name and length if sent
-    // @note: Add support for fileName & fileLength option in Schema & SDK.
-    //        The filepath property overrides filename and may contain a relative path.
-    if (typeof param.fileName === STRING) { (options || (options = {})).filename = param.fileName; }
-    if (typeof param.fileLength === 'number') { (options || (options = {})).knownLength = param.fileLength; }
-
+    if (param.fileName && typeof param.fileName === STRING) {
```

Comment:
What will happen when fileName is not present ?

Reply:
Not an issue, set as undefined.